test/jdk/sun/security/krb5/canonicalize/Test.java

```diff
@@ -25,18 +25,15 @@
  * @bug 6682516 8149521
  * @summary SPNEGO_HTTP_AUTH/WWW_KRB and SPNEGO_HTTP_AUTH/WWW_SPNEGO failed on all non-windows platforms
  * @modules java.security.jgss/sun.security.krb5
- * @run main/othervm  -Djava.security.krb5.conf=krb5.conf Test
+ * @run main/othervm -Djdk.net.hosts.file=${test.src}/TestHosts
+ *      -Djava.security.krb5.conf=krb5.conf Test
  */
 
-import java.net.InetAddress;
-import java.net.UnknownHostException;
 import sun.security.krb5.PrincipalName;
 
 public class Test {
     public static void main(String[] args) throws Exception {
         // This config file is generated using Kerberos.app on a Mac
-        String hostsFileName = System.getProperty("test.src", ".") + "/TestHosts";
-        System.setProperty("jdk.net.hosts.file", hostsFileName);
         System.setProperty("java.security.krb5.realm", "THIS.REALM");
```

Comment:
Maybe, you can move setting of `java.security.krb5.realm` and `java.security.krb5.kdc` properties to the `jtreg` header to have consistent approach of setting properties needed for a test run.

Reply:
Agree. Updated in latest commit, thanks